test/test_distributions.py

```diff
@@ -4439,6 +4527,7 @@ def f(*values):
             if Dist not in xfail:
                 self.assertTrue(any(n.isNondeterministic() for n in traced_f.graph.nodes()))
 
+    @skipIfRocm
```

Comment:
Can someone who understands this confirm this change is safe?

Reply:
cc: @bddppq

Reply:
Yeah, this is weird. If this were a new test, I would understand (it's not working on ROCm, fine, AMD will figure it out.) But this is a preexisting test...?

Reply:
No, I had forgotten I made that comment :) When I originally made the comment, I had thought your test was a newly added test. But it looks like it was a preexisting one, which means this change regress ROCm behavior. It may still be the right call to charge on here and let AMD fix it later. cc @iotamudelta here